db/db_impl/db_impl_open.cc

```diff
@@ -1053,6 +1053,7 @@ Status DBImpl::RecoverLogFiles(const std::vector<uint64_t>& log_numbers,
       versions_->MarkFileNumberUsed(max_log_number + 1);
       status = versions_->LogAndApply(cfd, *cfd->GetLatestMutableCFOptions(),
                                       edit, &mutex_);
+      TEST_SYNC_POINT_CALLBACK("DBImpl::Recover:AfterLogAndApply", nullptr);
```

Comment:
Should the name of the test sync point be "DBImpl::RecoverLogFiles:AfterLogAndApply"?